environments/india/inventory.ini.j2

```diff
@@ -50,6 +50,8 @@ elasticsearch_data=false
 {{ __kafka2__ }} kafka_broker_id=1 zookeeper_server_id=2
 {{ __kafka3__ }} kafka_broker_id=2 zookeeper_server_id=3
 {{ __rabbit2__ }}
+{{ __rabbit3__ }}
```

Comment:
I think these should be 
rabbit_a3
rabbit_c4

Reply:
sure prem i'll change this